backend/src/db-utils.js

```diff
@@ -305,6 +305,22 @@ const queryNewContractsCountAggregatedByDate = async () => {
   );
 };
 
+const queryUniqueContractsAggregatedByDate = async () => {
```

Comment:
Well, it is not unique contracts, but unique contract deployments. Unique contracts are the accounts. We can re-deploy different contracts to the same account, and this is what the query returns - the deployments.